src/app/Utils/KruizeUIConstants.ts

```diff
@@ -0,0 +1,2 @@
+export const KRUIZE_API_URL = process.env.KRUIZE_APP_API_URL || undefined;
+export const KRUIZE_API_PORT = process.env.KRUIZE_APP_API_PORT || undefined;
```

Comment:
**Suggestion:** It's better to provide a default value for the environment variables in case they are not set. This can prevent potential issues of these variables being undefined.
```suggestion
export const KRUIZE_API_URL = process.env.KRUIZE_APP_API_URL || 'http://localhost';
export const KRUIZE_API_PORT = process.env.KRUIZE_APP_API_PORT || '8080';
```